cypress/Shared/MeasureGroupPage.ts

```diff
@@ -175,7 +175,7 @@ export class MeasureGroupPage {
     //Supplemental data elements
     public static readonly supplementalDataDefinitionSelect = '[data-testid=supplementalDataElements]'
     public static readonly supplementalDataDefinitionDropdown = '#supplementalDataElements-listbox'
-    public static readonly supplementalDataDefinitionDescriptionTextBox = '[class="sc-iqcoie glofVe"]'
+    public static readonly supplementalDataDefinitionDescriptionTextBox = '[class="sc-evZas fJfawR"]'
```

Comment:
lets use the testid for this instead of the class name

Reply:
I ended up using the ID value, instead. I noticed that the data-testid values would reference the actual value that is selected for the supplemental data. For example, if you selected "Denom" for supplemental data then the data-testid value for the description field would be something like "data-testid=denom-description". I wanted to keep it generic so I used the id value which is "id=description".